lib/signals.c

```diff
@@ -185,6 +185,7 @@ signal_handler_init(void)
 	int sig;
 	struct sigaction act, oact;
 	int n;
+	static int remember = 1;
```

Comment:
I agree with you! (I was trying to suppress to minimal changes)
I'll fix so that it does not use a static variable.